### PR TITLE
Clean out `subscriptions` table only if replacement data is available

### PR DESCRIPTION
**Pull request by ikapelyukhin**
## Description

`subscriptions` table is cleaned out at the beginning of the sync process and populated at the very end. Any errors during the daily sync lead to this table ending up just empty.

Fixes bsc#1183615.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/183889/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 4ef37cc01e8508765e2244ac193384414fb3cac8 into f33b1166afafc06baaf402e40da01c3f9c56e435)

**Comment by suse-tests-fail** [comment]
Prophet reports failure. (Merged 4ef37cc01e8508765e2244ac193384414fb3cac8 into bd51887f819592d95ee624b9baff5c1b9c88c898)

**Review by digitaltom — APPROVED** [review]

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/184058/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 74810463a55de572e79757c0f4dd8d66e520ca84 into bd51887f819592d95ee624b9baff5c1b9c88c898)

File: package/obs/rmt-server.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Mar 16 19:53:15 UTC 2021 - Ivan Kapelyukhin <ikapelyukhin@suse.com>
+
+- Version 2.6.7
+- Clean out `subscriptions` table only if replacement data is already
+  available (bsc#1183615)
+
 Thu Feb 18 14:20:53 UTC 2021 - Natnael Getahun <natnael.getahun@suse.com>
```
[the remark this answers is not in the document]
Comment (digitaltom) [reply package/obs/rmt-server.changes:8]: I've already added it when solving a merge conflict in another pr.

[the remark this answers is not in the document]
Comment (ikapelyukhin) [reply package/obs/rmt-server.changes:8]: Cool, thanks.

**Pull request merged**